source/fundamentals/in-use-encryption.txt

```diff
@@ -1,854 +1,102 @@
-          partialFilterExpression={"keyAltNames": {"$exists": True}},
-      )
-
-      client_encryption = ClientEncryption(
-          kms_providers,
-          key_vault_namespace,
-          key_vault_client,
-          # The CodecOptions class used for encrypting and decrypting.
-          # This should be the same CodecOptions instance you have configured
-          # on MongoClient, Database, or Collection. We will not be calling
-          # encrypt() or decrypt() in this example so we can use any
-          # CodecOptions.
-          CodecOptions(),
-      )
-
-      # Create a new data key and json schema for the encryptedField.
-      data_key_id = client_encryption.create_data_key(
-          "local", key_alt_names=["pymongo_encryption_example_2"]
-      )
-      json_schema = {
-          "properties": {
-              "encryptedField": {
-                  "encrypt": {
-                      "keyId": [data_key_id],
-                      "bsonType": "string",
-                      "algorithm": Algorithm.AEAD_AES_256_CBC_HMAC_SHA_512_Deterministic,
-                  }
-              }
-          },
-          "bsonType": "object",
-      }
-
-      auto_encryption_opts = AutoEncryptionOpts(kms_providers, key_vault_namespace)
-      client = MongoClient(auto_encryption_opts=auto_encryption_opts)
-      db_name, coll_name = encrypted_namespace.split(".", 1)
-      db = client[db_name]
-      # Clear old data
-      db.drop_collection(coll_name)
-      # Create the collection with the encryption JSON Schema.
-      db.create_collection(
-          coll_name,
-          # uuid_representation=STANDARD is required to ensure that any
-          # UUIDs in the $jsonSchema document are encoded to BSON Binary
-          # with the standard UUID subtype 4. This is only needed when
-          # running the "create" collection command with an encryption
-          # JSON Schema.
-          codec_options=CodecOptions(uuid_representation=STANDARD),
-          write_concern=WriteConcern(w="majority"),
-          validator={"$jsonSchema": json_schema},
-      )
-      coll = client[db_name][coll_name]
-
-      coll.insert_one({"encryptedField": "123456789"})
-      print("Decrypted document: %s" % (coll.find_one(),))
-      unencrypted_coll = MongoClient()[db_name][coll_name]
-      print("Encrypted document: %s" % (unencrypted_coll.find_one(),))
-      try:
-          unencrypted_coll.insert_one({"encryptedField": "123456789"})
-      except OperationFailure as exc:
-          print("Unencrypted insert failed: %s" % (exc.details,))
-
-
-  if __name__ == "__main__":
-      main()
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
 
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: qe, csfle
 
-.. _explicit-client-side-encryption:
+Overview
+--------
 
-Explicit Encryption
-~~~~~~~~~~~~~~~~~~~
+You can use {+driver-short+} to encrypt specific document fields by using a
```

Comment:
I had to copy the text over from the sharedinclude file because that file reads "the {+driver-short+}", which doesn't work with "the PyMongo." More work will be needed to figure out how to use constants in this repo; for now, a copy/paste like this will work.